coverage/src/BamViewChart.js

```diff
@@ -505,8 +510,8 @@ function createBamView(bamHeader, data, element, bamViewControlsElement, broker)
 
             // Update the input fields
             bamViewControlsElement.querySelector('#bamview-region-chromosome').value = "chr" + orgChromosome;
```

Comment:
We should probably drop the "chr" here as well, since we want the UI to reflect the file.